### DISCUSSION TITLE
DevServer: how to specify a different content root?

### DISCUSSION THREAD

**Discussion by csvan** [webpack/webpack#17156] [General]
Scenario: 

I have 3 DevServer instances running on localhost:8081, localhost:8082, and localhost:8083. They all use ModuleFederation and serve an entrypoint called myEntry1.js, myEntry2.js etc.

Further, I have proxy server running at localhost:9090. It is supposed to delegate calls to one of the DevServer instances based on some internal logic.

Now, if I request for example localhost:9090/myEntry1.js, the proxy will retrieve the corresponding file from localhost:8081 just fine.

HOWEVER, when myEntry1.js in turn starts loading chunks, it does so by making calls DIRECTLY to localhost:8081, bypassing the proxy.

Is there any way to override this behavior, either to specify a specific origin, or to simply use the origin of the original request (e.g. localhost:9090 in this example)?

**Comment by alexander-akait** [comment]
Do you mean loading other chunks using `localhost:9090`?
